IFT-Explainer.md

```diff
@@ -69,6 +69,82 @@ Current subsetting does not address subsetting **design axes** (variable fonts).
 
 Changes to the Open Font Format or OpenType specifications are out of scope.
 
+## Incremental Font Transfer
+
+Web pages that wish to use an IFT font
+use the regular CSS [`@font-face`]() mechanism,
+and [opt-in](https://w3c.github.io/IFT/Overview.html#opt-in)
+with `tech(incremental)` or by using `font-tech(incremental)`
+inside an `@supports` rule.
+This allows the same CSS file to link to an IFT font
+and to some (larger, slower) fallback font.
+
+The link in the `src` descriptor points to an 
+[IFT font](https://w3c.github.io/IFT/Overview.html),
+which is a regular OpenType font

```

Comment:
Saying that an IFT font is a regular OpenType font may be confusing w.r.t. the actual encoding of data - the IFT payload is a regular font but it could be encoded as e.g. WOFF2.

Reply:
Perhaps. I felt it was more important to stress that this is not a new, competing format.

Reply:
Clarified.